chapters/handling-data.tex

```diff
@@ -141,92 +201,275 @@ \subsection{Research credibility}
-so agreeing on a protocol from the start of a project is ideal.
+	\url{https://about.citiprogram.org/en/series/human-subjects-research-hsr/}}
+
+In general, though, you shouldn't need to handle PII data very often
+once the data collection processes are completed.
+You can take simple steps to avoid risks by minimizing the handling of PII.
+First, only collect information that is strictly needed for the research.
+Second, avoid the proliferation of copies of identified data.
+There should only be one raw identified dataset copy
+and it should be somewhere where only approved people can access it.
+Finally, not everyone on the research team needs access to identified data.
+Analysis that requires PII data is rare
+and can be avoided by properly linking identifiers to research information
+such as treatment statuses and weights, then removing identifiers.
 
-In general, though, you shouldn't need to handle PII data very often.
-Once data is securely collected and stored, the first thing you will generally do is \textbf{de-identify} it.\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
+Therefore, once data is securely collected and stored,
+the first thing you will generally do is \textbf{de-identify} it.\sidenote{
+	\url{https://dimewiki.worldbank.org/wiki/De-identification}}
 \index{de-identification}
-De-identified data should avoid, for example, you being sent back to every household
-to alert them that someone dropped all their personal information on a public bus and we don't know who has it.
-This simply means creating a copy of the data that contains no personally-identifiable information.
+(We will provide more detail on this in the chapter on data collection.)
+This will create a working de-identified copy
+that can safely be shared among collaborators.
+De-identified data should avoid, for example,
+you being sent back to every household
+to alert them that someone dropped all their personal information
+on a public bus and we don't know who has it.
+This simply means creating a copy of the data
+that contains no personally-identifiable information.
 This data should be an exact copy of the raw data,
-except it would be okay for it to be publicly released.\cite{matthews2011data}
-Ideally, all machines used to store and process PII data are not only password protected, but also encrypted at the hard drive level
-(most modern operating systems provide such a tool).
-This means that even if you lose your computer with identifying data in it,
-anyone who gets hold of it still cannot access the information.
+except it would be okay if it were for some reason publicly released.\cite{matthews2011data}
+
+Note, however, that it is in practice impossible to \textbf{anonymize} data.
+There is always some statistical chance that an individual's identity
+will be re-linked to the data collected about them
+by using some other set of data that are collectively unique.
```

Comment:
"_colllectively unique_" - I understand this as I already know this, but I think it can be said in an easier way. I think the main issue is that we have not used the word _unique_ as a synonym of _identifying_ before. Those are sometimes synonyms but not always. Not all unique observations are identifying. Maybe write this instead:

_There is always some statistical risk that de-identified data can be re-identified by using other data that in combination with the de-identified date reveals the identity of some or all observerations._

Reply:
Reworded in 52b0751a5c9274478b7a8fdfa722b89af0317260

Reply:
can we somehow emphasize that it is the de-identified data set that can be re-identified. 

What about:

"[...] re-linked to the data collected about them, _even after de-identification_, by using some other set of data that [...]"

Reply:
Done in e0df2958287ef6fb0cef7e6f28e036401a936c91